### ISSUE TITLE
Move bnb/join to nodejs/join or nodejs/meet

### ISSUE THREAD

**Issue by bnb** [nodejs/admin#546]
I've gotten a bit tired of having to open up the GItHub issue for meetings just to access the join link, and I'd rather not fill up my bookmarks bar with random join links.

Inspired by a single page that @davidguttman made for the Mentorship meeting, I've built out what I suppose could be considered a tiny static site generator that automatically builds an index and individual redirect pages for every project defined in a JSON config file that can easily be deployed to GitHub Pages so we can have permalinks for each meeting.

Here's what the config looks like:

```
    {
      "name": "Technical Steering Committee (TSC)",
      "filename": "tsc",
      "link": "https://zoom.us/j/611357642"
    },
```

And it builds (where `nodejs` is the GItHub org and `join` is the repo name):

- A page called `tsc.html`, published to `/docs`. When used in conjunction with GitHub pages this would be published to `https://nodejs.github.io/join/tsc`. That link would redirect to whatever the value of `link` is in the object.
- A list entry in the `/docs/index.html` file so if someone goes to `https://nodejs.github.io/join` they can launch from there.

You can see this at work right now:
- Any of the following links work:
  - https://bnb.github.io/join
  - https://bnb.github.io/join/commcomm
  - https://bnb.github.io/join/tsc
  - https://bnb.github.io/join/modules
  - https://bnb.github.io/join/website-redesign

**Comment by mmarchini** [comment]
cc @nodejs/tsc @nodejs/community-committee

**Comment by bnb** [comment]
Updated slightly, accidentally did the wrong combo of shift and enter which created the issue a bit before it was fleshed out fully 👍🏻

**Comment by nschonni** [comment]
What about adding the links to https://nodejs.org/en/about/working-groups/

**Comment by mmarchini** [comment]
The page is _really_ long, as someone who can easily have 4+ foundation meetings in a week, having a single place with easy access seems really beneficial.

**Comment by bnb** [comment]
@nschonni a large part of the point of this is that it's automatically generated and can be easily accessed from a memorable URL in a repo that's easy to maintain. Including that in the working groups page makes less easy to access, harder to automate, and doesn't give both a focused place and a single, low-maintenance and memorable url per group.

Also, that page seems to be entirely unmaintained as Evangelism was moved to the CommComm and dechartered a year or two ago 😅

**Comment by nschonni** [comment]
Yup, i'm definitely not the target audience for this 😄 . Maybe I'll just re-frame my comment and suggest a more general, integrate it with the existing .org or .dev repos as a `/meet` instead of a new site

**Comment by mmarchini** [comment]
I think having a nodejs.org/nodejs.dev DNS entry is a really good idea, but keeping it separate and using GH Pages is probably simpler and more approachable than sharing code with the website repositories (and is easier to automate with Actions if we want to). Note that we already have precedence for this, https://benchmarking.nodejs.org/ is not a part of https://github.com/nodejs/nodejs.org, instead it is dyanmically generated by https://github.com/nodejs/benchmarking/.

**Comment by mmarchini** [comment]
Also worth noting that there's nothing preventing us from moving it to the org and later moving it to the website repos if we find that useful/easier to maintain/easier for the target audience to contribute.

**Comment by bnb** [comment]
I'm not sure what the barrier to entry would be for something like `join.nodejs.org` or `meet.nodejs.org` but I've got zero issue with that. I'm happy with that or .github.io, but would definitely want to not stall on that ❤️

**Comment by mmarchini** [comment]
> I'm not sure what the barrier to entry would be for something like join.nodejs.org or meet.nodejs.org but I've got zero issue with that

Creating a DNS entry is entirely separate from where the code will live, so it should have no influence on entry barrier

**Comment by MylesBorins** [comment]
+1 on creating a DNS entry and publishing via pages. 

Here is an example github pages deploy action if that is helpful https://github.com/tc39/proposal-top-level-await/blob/master/.github/workflows/publish.yml

**Comment by bnb** [comment]
Sounds like we have rough +1s for DNS. In that case, I'd like to propose `join.nodejs.org`. The URLs would then be `join.nodjes.org` for the index and `join.nodejs.org/<filename>` for each entry, assuming GitHub Pages doesn't require file extensions when you're doing a CNAME for non-index pages (it doesn't do this for `.github.io`)

@MylesBorins I'll work on getting that set up in my repo before we transfer (should be able to get it done today) 👍🏻
